src/header/classes.scss

```diff
@@ -1,11 +1,73 @@
 // Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import './index';
+@import '../breadcrumbs/index';
 
 .iui-page-header {
   @include iui-page-header;
+  @include iui-page-header-slim;
```

Comment:
I would suggest moving the modifier out of the mixin and putting it here like this:

```scss
.iui-page-header {
  @include iui-page-header;

  &.iui-slim {
    @include iui-page-header-slim;
  }
}
```

If you make the name more specific, you can even get rid of the `&`
```scss
.iui-page-header {
  @include iui-page-header;
}

.iui-page-header-slim {
  @include iui-page-header-slim;
}
```

Reply:
@veekeys had a similar comment.  [This was my reasoning behind it](https://github.com/iTwin/iTwinUI/pull/640#discussion_r877207632), but now that we've had 2 comments on the same thing I'll make the change.